packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/include/video_player_avfoundation/FVPDisplayLink.h

```diff
@@ -21,6 +21,11 @@
 /// The time interval between screen refresh updates.
 @property(nonatomic, readonly) CFTimeInterval duration;
 
+@end
+
+// An implementation of FVPDisplayLink using CADisplayLink.
+@interface FVPCADisplayLink : NSObject <FVPDisplayLink>
```

Comment:
Nit: It's hard to see the difference between `FVPCADisplayLink` and `FVPCVDisplayLink`. Could we name it `FVPCoreAnimationDisplayLink`? That seems outrageously long. If not, what you have is fine.